dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AnalyticsController.java

```diff
@@ -86,7 +86,7 @@
     // -------------------------------------------------------------------------
 
     @PreAuthorize( "hasRole('F_PERFORM_MAINTENANCE')" )
-    @GetMapping( value = RESOURCE_PATH + ANALYZE_PATH, produces = { APPLICATION_JSON_VALUE, "application/javascript" } )
+    @GetMapping( value = RESOURCE_PATH + EXPLAIN_PATH, produces = { APPLICATION_JSON_VALUE, "application/javascript" } )
     public @ResponseBody Grid getAnalyzeJson( // JSON
```

Comment:
Should we also change the name of the methods and any exiting variable, and use`explain` instead of `analyze`?

Reply:
yeah, good idea, let me change it